patches/minecraft/net/minecraft/tags/TagManager.java.patch

```diff
@@ -39,3 +17,11 @@
              SerializationTags.m_13202_(tagcontainer);
              this.f_13478_ = tagcontainer;
           }
+@@ -73,6 +_,7 @@
+          }, p_144577_);
+          return new TagManager.LoaderInfo<>(p_144578_, completablefuture);
+       } else {
++         if (true) return net.minecraftforge.common.ForgeTagHandler.makeCustomTagTypeLoader(p_144578_, p_144576_, p_144577_, customTagTypeReaders);
```

Comment:
This seems to be silencing a vanilla warning. While I agree if we are able to make a custom tag type loader we shouldn't warn that it isn't found, but we probably should still have a warning if it failed (be it that it couldn't find the registry or that the registry doesn't support custom tag types).